src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task {
+    public Deadline(String description, String date) {
+        this.description = description;
+        this.type = "D";
+        this.date = parseDate(date);
+        this.isDone = false;
```

Comment:
Maybe consider using the parent constructor (using super) to setup the inherited members. That way you show which members were inherited, and which are specific to the child class.